activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1176,13 +1188,20 @@ def update_table_definition(table_name, base) #:nodoc:
       end
 
       def add_index_options(table_name, column_name, comment: nil, **options) # :nodoc:
+        if options.key?(:includes)
+          if ActiveRecord::Base.connection.select_value("SELECT CURRENT_SETTING('server_version_num')::INT < 110000")
```

Comment:
You can get PostgreSQL version by using `database_version` like https://github.com/rails/rails/blob/01336b71af6ac798e48101b4ceb2de00397243aa/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L204-L206

Reply:
Didn't think there was a method for this. I just pushed the commit with the changes.

Thanks for the feedback!